modules/travel_pay/spec/services/token_service_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe TravelPay::TokenService do
+  context 'get_tokens' do
+    let(:user) { build(:user) }
+    let(:tokens) do
+      {
+        veis_token => 'veis_token',
+        btsss_token => 'btsss_token'
+      }
+    end
+    let(:tokens_response) do
+      Faraday::Response.new(
+        body: tokens
+      )
+    end
+
+    before do
+      allow_any_instance_of(TravelPay::TokenClient)
+        .to receive(:get_tokens)
+        .with(user)
+        .and_return(tokens_response)
+    end
```

Comment:
Let's add some `it` blocks/expectations here. I think the services should be responsible for handling errors. Something I might expect to see in the tests are things like:

- what happens if one token request fails? What does/should the response look like? What is/should be the behavior in cases like this?
- do both requests get made in the happy path?

and any others you can think of